_platforms/xnat.md

```diff
@@ -5,14 +5,30 @@ title: XNAT
 platform-image: /assets/img/platforms/xnat_logo.png
 ---
 
-[XNAT](https://xnat.org/) is an open source imaging informatics platform capable of storing and managing medical images and associated data. Developed by the Neuroinformatics Research Group at Washington University, XNAT can be used to support a wide range of imaging-based projects.
+[XNAT](https://xnat.org/) is an open-source imaging informatics platform for storing and managing medical 
+images and associated data for research purposes. At GSTT we used XNAT to retrieve, anonymise and manage medical imaging 
+and non-imaging data for purposes of AI development, AI evaluation, and for data provision for various research projects 
+which are run at GSTT or in collaboration with GSTT. 
 
-Data is ingested from the Picture Archiving and Communications Systems (PACS), the industry-standard storage system for different medical imaging types, into XNAT where it is anonymised (or more accurately, de-identified) and sorted into relevant projects, which ensures the data:
+XNAT at GSTT is currently connected to 3 PACS systems (radiology, foetal cardiology and radiotherapy PACS). Data anonymisation
+processes include DICOM meta-data removal, face blurring and removal of burnt-in data. Data used for AI evaluation are pushed 
+to relevant DICOM nodes using a Python script; data used for AI development are fetched by [MLOPs](https://gstt-csc.github.io/platforms/mlops.html) via
+XNAT's REST-API, and data are sent to external collaborators via pathways agreed in ethics approval of each individual project.
 
-- Is only visible to those who need it
-- Can be deleted upon project completion
-- Is compliant with the [DICOM Standards Supplement 142](https://www.dicomstandard.org/News-dir/ftsup/docs/sups/sup142.pdf)
+<figure>
+<img src="https://github.com/GSTT-CSC/XNAT/blob/main/assets/anon-flowchart.jpg?raw=true" width="100%">
+<figcaption>Schematic representation of XNAT setup at GSTT.</figcaption>
```

Comment:
Is there a way to make front size of the figure caption smaller (or italicised)? At first glance, it's easy to not notice the distinction between that and the body text.

![Screenshot 2023-04-11 at 10 11 52](https://user-images.githubusercontent.com/67117138/231113639-1a3357ad-8aae-4702-accd-6e5c5c62a6d8.png)